profiles/lib/way_handlers.lua

```diff
@@ -521,7 +521,7 @@ function WayHandlers.blocked_ways(profile,way,result,data)
     local construction = way:get_value_by_key('construction')
 
     -- Of course there are negative tags to handle, too
-    if construction and construction ~= 'no' and construction ~= 'widening' then
+    if construction and construction ~= 'no' and construction ~= 'widening' and construction ~= 'minor' then
```

Comment:
how about doing an explicit check for the tags we wan't to block from routing? Otherwise we might just end up excluding another strange tag we might not yet even know about/that is introduced in the future.